ci/Dockerfile.test

```diff
@@ -0,0 +1,14 @@
+FROM {{ base_image.image }}
+
+COPY hail/python/setup-hailtop.py /hailtop/setup.py
+COPY hail/python/hailtop /hailtop/hailtop/
+RUN python3 -m pip install --no-cache-dir /hailtop \
+  && rm -rf /hailtop
```

Comment:
Since you need hailtop, you might just build off service_base_image which has hailtop, gear and web_common installed.